modules/ipc/apps/zenoh_topic_echo.cpp

```diff
@@ -21,9 +22,35 @@
 #include "zenoh_program_options.h"
 
 namespace heph::ipc::apps {
+
+namespace {
+constexpr auto MAX_ARRAY_LENGTH = 100;
```

Comment:
maybe this could be a paramter? 100 seems like a lot.

we could have a flag: `--arr-max-len` with default value 0 or -1 for infinite

Reply:
how about this? the benefit is that it supports similar usage as with ROS...